test/routable_component.js

```diff
@@ -34,22 +34,22 @@ class MyController {
 
 /** @test {RoutableComponent} */
 describe('RoutableComponent', function() {
-  lazy('routes', function() { return new RoutableComponentRoutes(this.routing_classes) });
-  lazy('routing_classes', function() { return MyUsefulRouting });
+  lazy('routes', function() { return new RoutableComponentRoutes(this.routingClasses) });
+  lazy('routingClasses', function() { return MyUsefulRouting });
   lazy('components', function() { return {MyEventSourceComponent: new MyEventSourceComponent()} });
-  lazy('controller_classes', function() { return {MyController} });
+  lazy('controllerClasses', function() { return {MyController} });
```

Comment:
__[eslint]__
* Error - 'lazy' is not defined. (no-undef)
* Warning - Unexpected function expression. (prefer-arrow-callback)
* Error - Missing semicolon. (semi)

<!-- Comment by [SideCI](https://sideci.com) -->